.github/scripts/svgo.config.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+  plugins: [
+    {
+      name: 'preset-default'
+    },
+    {
+      name: 'removeDimensions',
+      params: {
+        overrides: {
+          removeDimensions: true,
+        }
+      }
+    },
+    {
+      name: 'removeScriptElement',
+      params: {
+        overrides: {
+          removeScriptElement: true,
+        }
+      }
+    },
+    {
```

Comment:
Shouldn't we remove strokes as well?

Reply:
No. The reason is that this might change the icon. In such cases we should instead tell the user that the stroke needs to be removed in another github workflow.

Relevant PR comment: https://github.com/devicons/devicon/pull/1554#discussion_r1030321954